src/main/java/nl/tudelft/ti2206/group9/gui/SettingsScene.java

```diff
@@ -37,7 +35,7 @@
      * @return whether sound is enabled.
      */
     public static boolean isSoundEnabled() {
-    	return sound;
+    	return State.isSoundEnabled();
```

Comment:
If you make it return State.isSoundEnabled(), why does this method still exist?

<interface,review